### ISSUE TITLE
Can explore_all()  function allow more than 1 single target= variable?

### ISSUE THREAD

**Issue by sfd99** [rolkra/explore#40]
**_Hi Roland,_** 

Thanks for the update 
to the _GREAT_ explore PKG.

Your work and dedication
is wonderful work 
_(as always..._) ! .

**Q:**
Can  the explore_all()  function 
allow more than _1 single_ target variable?.

**EX:**
penguins %>% 
    explore_all( target = species   **_AND  island_** )

The output report / plots
would be longer 
than for using 
just 1 single target= var.

**BUT...**
I can see applications
of explore_all(),
where allowing MORE THAN 1 single var as the target= 
would be _VERY powerful_
for analysis.

**sfd99
San Francisco
explore PKG 1.1.0
Ubuntu Linux 20.04,
latest Rstudio and R.**

**Comment by rolkra** [comment]
Interesting, I never thought about this.

I see cases (and I had them by myself) that exploring data using 2 or more targets is useful.

In this cases I simply call explore() or report() with each target. 

You could even create your own notebook using
create_notebook_explore() and then copy paste the target section and adding a new target. So you would have both targets in one HTML report.

**Comment by sfd99** [comment]
Ah, yes...
I see your point.
But this would still
consider the 2 targets _individually_.
(not combined into a **single** target).

But it would be powerful
to see _one_ density plot only,
ie: 
value distribution for
   body_mass_g
for a **_combined_** target:
"Adelie" pengs
only living in "Dream" island.

If you develop the idea,
let me know
and I'll help test it. :-)

Anyhow,
hope you are having
a nice weekend in wonderful Wien. :-)

**Comment by rolkra** [comment]
Hmmm
You could define a new target variable

library(tidyverse)
library(explore)
data <- use_data_penguins()
data <- data |> 
  mutate(adelie_dream = ifelse(species == "Adelie" & island == "Dream", 1, 0))
data |> 
  explore_all(target = adelie_dream)
data |> 
  select(-species, -island) |> 
  explain_tree(target = adelie_dream)

**Comment by rolkra** [comment]
Yes, I guess the new feature would be that explore() function accepts as target a variable OR an logical expression (that returns 0/1 and this is automatically the target)
Thanks!

**Comment by sfd99** [comment]
...OR an logical expression (that returns 0/1 and this is automatically the target).
yes, the above sounds easier to implement.
